Source/JavaScriptCore/runtime/JSGlobalObject.cpp

```diff
@@ -1416,6 +1416,9 @@ capitalName ## Constructor* lowerName ## Constructor = featureFlag ? capitalName
     GetterSetter* regExpProtoUnicodeGetter = getGetterById(this, m_regExpPrototype.get(), vm.propertyNames->unicode);
     catchScope.assertNoException();
     m_linkTimeConstants[static_cast<unsigned>(LinkTimeConstant::regExpProtoUnicodeGetter)].set(vm, this, regExpProtoUnicodeGetter);
+    GetterSetter* regExpProtoUnicodeSetsGetter = getGetterById(this, m_regExpPrototype.get(), vm.propertyNames->unicodeSets);
+    catchScope.assertNoException();
+    m_linkTimeConstants[static_cast<unsigned>(LinkTimeConstant::regExpProtoUnicodeSetsGetter)].set(vm, this, regExpProtoUnicodeSetsGetter);
```

Comment:
I think RegExpPrototype watchpoint optimization is probably missing. Please grep all use of existing getters carefully and align the new one with this.

Reply:
I grep'ed the other uses of the RegExp getters and only find uses in DFGAbstractInterpreterInlines.h and DFGFixupPhase.  The unicodeSets getter is already added to those sections.

Reply:
Fixed.